packages/documentation-site/patternfly-docs/content/get-started/about-patternfly.md

```diff
@@ -0,0 +1,363 @@
+<br />
+
+#### New communities
+Sometimes Flyers branch out and build groups of their own, creating new communities. While these communities are separate from PatternFly and not supported as part of our design system, we still love seeing our Flyers turning their visions into actions.
+
+- **[PatternFly Elements](https://patternflyelements.org):** A community created by web-based developers at Red Hat, focused on creating web components for use across Red Hat's sites and SaaS products. It offers theming options for your own brand library.
+
+- **[PatternFly Kotlin](https://github.com/patternfly-kotlin/patternfly-kotlin):** A Kotlin implementation of PatternFly based on fritz2, targeting Kotlin/JS. The goal of this project is to provide all PatternFly components in Kotlin, matching the reactive nature of fritz2. The components use stores, handlers, and other elements from the fritz2 API. For a quick overview, [view the PatternFly Kotlin showcase.](https://patternfly-kotlin.github.io/patternfly-kotlin-showcase/#home)
+
+- **[Ansible Component Guide and Sketch Library](https://www.sketch.com/s/6ccbd710-267d-4b69-9dae-bc19e1551056):** An additional resource that designers can use that is built on top of existing PatternFly components. It is a rapid mockup prototyping tool that can be used to quickly put together repeatable design patterns and layouts across projects. This is specific to Ansible, but many of the components are generalized and can fit many product use cases.
+
+- **[PatternFly for Yew](https://github.com/patternfly-yew/patternfly-yew)**: Provides PatternFly components for Yew—a Rust based framework for creating web applications that can run in the browser using WASM. The project aims to create Yew components for all components and concepts found in PatternFly. For a quick demo and starter template, [see the PatternFly Yew Quickstart.](https://github.com/patternfly-yew/patternfly-yew-quickstart)
+
+## Where do I start?
+
+Now that you've been introduced to PatternFly, you're ready to start designing or start developing your product. Looking to get involved in the behind-the-scenes work? Check out our contribution guidelines for more instruction.
+
+<br />
+<Grid sm={12} md={4} hasGutter>
+<Button size="lg" variant="secondary" component="a" href="/get-started/design"> Start designing </Button>
+<Button size="lg" variant="secondary" component="a" href="/get-started/develop"> Start developing </Button>
+<Button size="lg" variant="secondary" component="a" href="/get-started/contribute/contributing-to-patternfly"> Contribute to PatternFly</Button>
+</Grid>
+<br />
+
+## Release cadence
+
+We plan to release a new major version of PatternFly every year. These major releases will be the only releases with planned "breaking" changes that alter core code functionality. Along with each major release, we will provide detailed upgrade guides and codemods to support the work needed to upgrade your products to the latest version of PatternFly.
```

Comment:
`"breaking" changes that alter core code functionality` - since we use "core" as the css repo, should this be something like `"breaking changes" that alter the way your code interacts with the PatternFly code` or something like that? Or say PatternFly's API but it's not an API like some may define it.

Reply:
great point! I like that suggestion